3rdParty/OUIDataset/create_oui_data.py

```diff
@@ -4,9 +4,13 @@
 from typing import Optional
 import urllib.request
 
-MANUF_URL = "https://gitlab.com/wireshark/wireshark/-/raw/master/manuf"
+MANUF_URL = "https://gitlab.com/wireshark/wireshark/-/raw/master/epan/manuf-data.c"
 MANUF_FILENAME = "manuf.dat"
 
+MA_L = 0
+MA_M = 1
+MA_S = 2
```

Comment:
We these consts used anywhere?

Reply:
They are required for `eval` function. During the processing for lines https://gitlab.com/wireshark/wireshark/-/blob/master/epan/manuf-data.c?ref_type=heads#L18 it raises an error `Undefined variable MA_S`. To keep changes minimal instead of parsing this array to detect mask value I used MAC length (character length shows mask for 24/28/36) based detection

Reply:
got it. That's the downside of using `eval()`, it's not easy to track what the code is really doing... maybe we should be more explicit, parse these lines as strings and convert them to python lists?

Reply:
It is possible, I tried a fast solution. I can take a look again